pkg/pusher/pusher.go

```diff
@@ -84,13 +89,14 @@ func (s *Service) chunksWorker() {
 			chunksInBatch++
 			s.metrics.TotalChunksToBeSentCounter.Inc()
 
-			t, err := s.tag.GetByAddress(ch.Address())
+			t, err := s.tag.Get(ch.TagID())
 			if err != nil {
-				s.logger.Debugf("pusher: get tag by address %s: %v", ch.Address(), err)
-				//continue // // until bzz api implements tags dont continue here
+				s.logger.Debugf("pusher: get tag by uid %s: %v", ch.Address(), err)
+				//continue // // until bzz api implements tags, dont continue here
```

Comment:
can we please remove this? also, i dont see why we should ever continue here. continue means the chunk will never get pushed, will never get set as synced, and will result in index leakages.